x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/index_priority_field.tsx

```diff
@@ -28,9 +28,11 @@ export const IndexPriorityField: FunctionComponent<Props> = ({ phase }) => {
 
   const initialToggleValue = useMemo<boolean>(() => {
     return (
+      // enable index priority if it's set
+      // enable index priority for new phases
```

Comment:
These comments don't seem to make sense in this order. Can we rearrange them? We could do something like this:

```
    return (
      // enable index priority for new policies:
      isNewPolicy ||
      // enable index priority for new phases:
      !policy.phases[phase]?.actions ||
      // enable index priority if it's set:
      policy.phases[phase]?.actions?.set_priority != null
    );
```

Reply:
Thanks for the feedback, I'm not sure how this got messed up. I'll make this change